CHANGELOG.md

```diff
@@ -5,12 +5,46 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
-## [Unreleased]
+## [2.05.0.1.0]
+
+### Added 
+- Implements anti-entropy protocol for querying transactions from other 
+  nodes' mempools. Before, nodes wouldn't sync mempool contents with one another.
+- Structured logging in the mining code paths. This will shine light 
+  on what happens to transactions (successfully added, skipped or errored) that the
+  miner considers while buildings blocks.
+- Added the mined microblock event, which includes information on transaction
+  events that occurred in the course of mining (will provide insight
+  on whether a transaction was successfully added to the block,
+  skipped, or had a processing error).
+- For v2 endpoints, can now specify the `tip` parameter to `latest`. If 
+  `tip=latest`, the node will try to run the query off of the latest tip. 
+- Adds the /v2/headers endpoint, which returns a sequence of SIP-003-encoded 
+  block headers and consensus hashes (see the ExtendedStacksHeader struct that 
+  this PR adds to represent this data).
+- Adds the /v2/data_var endpoint, which returns a contract's data variable 
+  value and a MARF proof of its existence.
 
 ### Changed
-
 - The pessimistic execution cost estimator differentiates between
   contracts with different origin addresses.
+- Updated the mined block event. It now includes information on transaction 
+  events that occurred in the course of mining (will provide insight
+  on whether a transaction was successfully added to the block, 
+  skipped, or had a processing error).
+- Updated some of the logic in the block assembly for the miner and the follower
+  to consolidate similar logic. Added functions `setup_block` and `finish_block`.
+- Makes the p2p state machine more reactive to newly-arrived 
+  `BlocksAvailable` and `MicroblocksAvailable` messages for block and microblock 
+  streams that this node does not have. If such messages arrive during an inventory 
+  sync, the p2p state machine will immediately transition from the inventory sync 
+  work state to the block downloader work state, and immediately proceed to fetch 
+  the available block or microblock stream.
+
+### Fixed 
+- Updates the lookup key for contracts in the pessimistic cost estimator. Before, contracts
+  published by different principals with the same name would have had the same 
+  key in the cost estimator.
```

Comment:
This is the same update as the one described at line 29, but it's better described here, so you can delete the entry at line 29.

Reply:
oh yup good catch.